web/src/main/java/org/openmrs/web/Listener.java

```diff
@@ -172,8 +166,7 @@ public void contextInitialized(ServletContextEvent event) {
 				
 				//ensure that we always log the runtime properties file that we are using
 				//since openmrs is just booting, the log levels are not yet set. TRUNK-4835
-				Logger.getLogger(getClass()).setLevel(Level.INFO);
```

Comment:
How about setting the log level?

Reply:
I think there is no setLevel in slf4j, it is replaced with info() method call

Reply:
So how are log levels set?

Reply:
I think slf4j doesn't allow dynamic level changes. Maybe it depends on implementation

Reply:
Can we then revert to the original one which allowed setting the log level?

Reply:
@dkayiwa  Fixed :)

Reply:
Ticket status says you are still doing more work on this. 😊

Reply:
Done :)